mdata/cache/sort_uint32.go

```diff
@@ -0,0 +1,7 @@
+package cache
+
+type sortableUint32 []uint32
```

Comment:
a go best practice is to name types for sorting after what order the sorting happens in.
e.g. https://github.com/raintank/metrictank/blob/38dd0f4f783919a63da0143977ec539564c30f57/mdata/aggregator.go#L23
(we don't apply this everywhere throughout the code yet, but we should)

so this could be called uint32Asc for example